.gitignore

```
@@ -1,7 +1,9 @@
 .DS_Store
 public
+build-bloom-command-json
 build-command-docs
 build-command-json
```

```suggestion
build-command-json
build-json-command-json
```
Is the name of the other JSON command.